include/nbl/asset/utils/CDirQuantCacheBase.h

```diff
@@ -277,8 +277,9 @@ class CDirQuantCacheBase : public impl::CDirQuantCacheBase
 
 			auto& particularCache = std::get<cache_type_t<CacheFormat>>(cache);
 			cache_type_t<CacheFormat> backup;
+
 			if (!replaceCurrentContents)
-				backup.swap(std::move(particularCache));
+				backup.swap(particularCache);
```

Comment:
why no more move?

Reply:
because you would encounter awful compile error https://en.cppreference.com/w/cpp/container/unordered_map/swap

Reply:
ah ok, `swap` is dont via references